src/components/sub/menu/menu-bar.jsx

```diff
@@ -1,134 +1,145 @@
+    //   action: () => editor.chain().focus().toggleHeading({ level: 1 }).run(),
+    //   isActive: () => editor.isActive("heading", { level: 1 }),
+    // },
+    // {
+    //   icon: "h-2",
+    //   title: "Heading 2",
+    //   action: () => editor.chain().focus().toggleHeading({ level: 2 }).run(),
+    //   isActive: () => editor.isActive("heading", { level: 2 }),
+    // },
+    // {
+    //   icon: "paragraph",
+    //   title: "Paragraph",
+    //   action: () => editor.chain().focus().setParagraph().run(),
+    //   isActive: () => editor.isActive("paragraph"),
+    // },
+    // {
+    //   icon: "list-unordered",
+    //   title: "Bullet list",
+    //   action: () => editor.chain().focus().toggleBulletList().run(),
+    //   isActive: () => editor.isActive("bulletList"),
+    // },
+    // {
+    //   icon: "list-ordered",
+    //   title: "Ordered list",
+    //   action: () => editor.chain().focus().toggleOrderedList().run(),
+    //   isActive: () => editor.isActive("orderedList"),
+    // },
+    // {
+    //   icon: "list-check-2",
+    //   title: "Task list",
+    //   action: () => editor.chain().focus().toggleTaskList().run(),
+    //   isActive: () => editor.isActive("taskList"),
+    // },
+    // {
+    //   icon: "code-box-line",
+    //   title: "Code block",
+    //   action: () => editor.chain().focus().toggleCodeBlock().run(),
+    //   isActive: () => editor.isActive("codeBlock"),
+    // },
+    // {
+    //   type: "divider",
+    // },
+    // {
+    //   icon: "double-quotes-l",
+    //   title: "Blockquote",
+    //   action: () => editor.chain().focus().toggleBlockquote().run(),
+    //   isActive: () => editor.isActive("blockquote"),
+    // },
+    // {
+    //   icon: "separator",
+    //   title: "Horizontal rule",
+    //   action: () => editor.chain().focus().setHorizontalRule().run(),
+    // },
+    // {
+    //   type: "divider",
+    // },
+    // {
+    //   icon: "text-wrap",
+    //   title: "Hard break",
+    //   action: () => editor.chain().focus().setHardBreak().run(),
+    // },
+    // {
+    //   icon: "format-clear",
+    //   title: "Clear format",
+    //   action: () => editor.chain().focus().clearNodes().unsetAllMarks().run(),
+    // },
+    // {
+    //   type: "divider",
+    // },
+    // {
+    //   icon: "arrow-go-back-line",
+    //   title: "Undo",
+    //   action: () => editor.chain().focus().undo().run(),
+    // },
+    // {
+    //   icon: "arrow-go-forward-line",
+    //   title: "Redo",
+    //   action: () => editor.chain().focus().redo().run(),
+    // },
+  ];
 
   return (
     <div className="editor__header">
       {items.map((item, index) => (
         <Fragment key={index}>
-          {item.type === 'divider' ? <div className="divider" /> : <MenuItem {...item} />}
+          {item.type === "divider" ? (
+            <div className="divider" />
+          ) : (
+            <MenuItem {...item}>
+              <item.icon />
+            </MenuItem>
+          )}
```

Comment:
Bc i changed MenuItem to render props.children, im rendering the react component set on the icon field

eg: icon = RiBold is the component and <RiBold/> is the react Element being generated when <item.icon /> is rendered.